docs/components/oculus-touch-controls.md

```diff
@@ -34,34 +35,43 @@ mappings, events, and a Touch controller model.
 | triggerup            | Trigger released.              |
 | triggertouchstart    | Trigger touched.               |
 | triggertouchend      | Trigger no longer touched.     |
+| triggerchanged       | Trigger changed.               |
```

Comment:
FYI: Firefox ships with an implementation of experimental Gamepad events: `gamepadbuttondown`, `gamepadbuttonup`, and `gamepadaxismove` (fired on `window`).

If you're curious, might want to check these links out:

* https://github.com/w3c/gamepad/issues/4
* https://github.com/w3c/gamepad/issues/56
* https://github.com/w3c/gamepad/pull/15